### PR TITLE
Blog Post - English <Path to Industry 4.0>

### PR DESCRIPTION
**Pull request by shripaprikar**
Added New Blog Post - English <Path to Industry 4.0>, under Gas & Oil Category... Used existing image, we can update image later once received

### CODE REVIEW

File: blogs/index.php [hunk 1]
```diff
@@ -477,7 +477,19 @@ class="title">Shining Up a Rusty Industry with Artificial Intelligence</a>
                             </div>
                         </div>
 
-
+                        <div class="col-md-6 mix OilGas">
+                            <div class="blogBox">
+                                <div class="blogPic">
+                                    <div class="industiesText">
+                                        Oil & Gas
```
Comment (Sumeshks29) [inline blogs/index.php]: @shripaprikar This blog shouldn't come under Oil and Gas Section, please confirm with @Poorvasisodia. Also any new blog must be on top, unless the marketing team has asked specifically to put it in the bottom.

@Poorvasisodia Please provide a relevant image.

File: blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    <script src="../js/flickity.pkgd.min.js"></script>
+    <script src="../js/readmore.js"></script>
+    <script src="../js/counting.js"></script>
+    <script src="../js/script.js"></script>
+    <script src="../js/parallax.js"></script>
+
+
+</body>
+
+</html>
```
Comment (Sumeshks29) [inline blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php:200]: @shripaprikar please add an empty line in the end.

**Review by Sumeshks29 — CHANGES_REQUESTED** [review]

File: blogs/index.php (hunk 1 above)
Comment (shripaprikar) [reply blogs/index.php]: Category has been already changed in index page, its not under oil & gas

File: blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php (hunk 2 above)
Comment (shripaprikar) [reply blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php:200]: Do you mean to remove the empty line? Or need to add after HTML tag?

File: blogs/index.php (hunk 1 above)
Comment (shripaprikar) [reply blogs/index.php]: Added under Group : Data scientists, CDO.

File: blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php (hunk 2 above)
Comment (Sumeshks29) [reply blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php:200]: you need to add one extra empty line and html tag

File: blogs/index.php (hunk 1 above)
Comment (Sumeshks29) [reply blogs/index.php]: @shripaprikar what about the image?

Comment (shripaprikar) [reply blogs/index.php]: I am waiting for image... still waiting for that

File: blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php (hunk 2 above)
Comment (shripaprikar) [reply blogs/best-practices-to-follow-for-data-science-projects-in-manufacturing-industry.php]: Added Extra line space

**Comment by Poorvasisodia** [comment]
> Added New Blog Post - English <Path to Industry 4.0>, under Gas & Oil Category... Used existing image, we can update image later once received

@shripaprikar I have shared the image yesterday.

File: blogs/index.php (hunk 1 above)
Comment (Poorvasisodia) [reply blogs/index.php]: @shripaprikar I have shared the relevant image yesterday. Please check

Comment (shripaprikar) [reply blogs/index.php]: Used existing image

Comment (Sumeshks29) [reply blogs/index.php]: futuristic-industry-4-0-engineering-concept (1)
Please compress this image to the lowest size possible and use it.

**Review by Sumeshks29 — APPROVED** [review]

**Pull request merged**